internal/apps/ecs.go

```diff
@@ -172,7 +176,7 @@ func (e *ecs) Deploy(sg terminal.StepGroup, ui terminal.UI) error {
 	s = sg.Add("%s: deploying app container...", e.Name)
```

Comment:
Should we add more logs under the spinner?

```
⠼ squibby: deploying app container...                                                                                                   
 │ Updated ECS task definition to ...
 │ Waiting for the app to become healthy
 ```

Reply:
@AutomationD, I suggest creating a new story for this proposal with a detailed description because the ECS deployment feature is not complete